ipn/ipnlocal/local.go

```diff
@@ -1635,6 +1635,38 @@ func (b *LocalBackend) authReconfig() {
 				set(peer.Name, peer.Addresses)
 			}
 		}
+
+		addFallback := func() {
+			for _, resolver := range nm.DNS.FallbackResolvers {
+				res, err := parseResolver(resolver)
+				if err != nil {
+					b.logf(err.Error())
```

Comment:
the first parameter to logf is a log format, not a string.

so the obvious fix is `logf("%s", err)` but then you get rate-limited on `"%s"` (sigh). So invent some prefix?